sdk/src/main/java/ch/admin/bag/covidcertificate/eval/models/DccHolder.kt

```diff
@@ -26,25 +28,28 @@ data class DccHolder(
 	var certType: CertType? = null
 		internal set
 
+	fun isFullCertificate() = euDGC != null
```

Comment:
In that case (euDcc is kept) this function does not make sense

Reply:
Also, we wouldn't need to make euDGC nullable.

Reply:
As written in another comment above: The DccHolder is a holder object of a single certificate. While it's true that we always store the EU cert and optionally a cert light, the DccHolder is the decoded QR code data, which is either an EU cert or a cert light.
Otherwise the certificate decoding mechanism has to receive two input strings, one for the EU QR Code and one for the QR Code light.

Reply:
True. Then, do we want to have something like this as a sanity check?
```kotlin
init {
     if (!(euDgc != null ^ dccLight != null)){
        throw IllegalStateException("Must set either euDgc xor dccLight)
     }
}
```

Reply:
Makes sense. I will add that